### DISCUSSION TITLE
What is the difference between free and open content?

### DISCUSSION THREAD

**Discussion by Trilarion** [opengaming/osgameclones#1428] [Q&A] [answered]
The content field for the games are one of 'commercial', 'free', 'open', 'swappable', but what is the difference between free and open?

**Comment by Trilarion** [comment]
Related to #592

**Comment by cxong** [comment, marked as answer]
https://github.com/opengaming/osgameclones/commit/9d72fbf89351fba76d46c3d732908f8bed6bb23c

**Comment by cxong** [comment]
#373

**Comment by Trilarion** [comment]
Ah, the grey asterisk footnote completely escaped my attention. Sorry. :)
